api/rest/client/request.go

```diff
@@ -58,10 +58,8 @@ func (c *Client) doStreamRequest(method, path string, body io.Reader, headers ma
 		r.Header.Set(k, v)
 	}
 
-	// Here are the streaming specific modifications
-	// Using HTTP 2.0 to enable parallel reading req and writing resp
-	r.ProtoMajor = 2
-	r.ProtoMinor = 0
+	r.ProtoMajor = 1
+	r.ProtoMinor = 1
```

Comment:
what was the issue here for not maintaining http/2.0

Reply:
I couldn't get simultaneous streaming of request and response working which was the whole reason I moved to 2.0 in the first place.  I could bring it back if you like but I couldn't think of a reason to keep it so went back to the standard choice.

Reply:
SGTM